mne_bids/write.py

```diff
@@ -939,6 +1096,28 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
         bids_raw_folder = bids_fname.split('.')[0]
         bids_fname = op.join(bids_raw_folder, bids_fname)
 
+    # Anonymize
+    if anonymize is not None:
+        if 'daysback' not in anonymize or anonymize['daysback'] is None:
+            raise ValueError('`daysback` argument required to anonymize.')
+        daysback = anonymize['daysback']
+        daysback_min, daysback_max = _get_anonymization_daysback(raw)
+        if daysback < daysback_min:
+            warn('`daysback` is too small; the measurement date ' +
+                 'is after 1925, which is not recommended by BIDS.' +
+                 'The minimum `daysback` value for changing the measurement' +
+                 'date of this data to before this date is %i' % daysback_min)
+        if daysback > daysback_max:
+            raise ValueError('`daysback` exceeds maximum value MNE is able ' +
+                             'to store, must be less than %i' % daysback_max)
```

Comment:
this is only relevant if you're going to store in fif.

Reply:
Good point, I thought that it would fail since meas_date is stored in int32 but it passed and was cast to an int. Must have been some smart coding by @bloyl